plugins/external/discover_test.go

```diff
@@ -6,11 +6,15 @@ package plugins
 import (
 	"testing"
 
+	"github.com/odpf/meteor/registry"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestDiscoverPlugins(t *testing.T) {
```

Comment:
@mabdh I need some context arounf `plugins/external` directory, around its usecase, and requirements. Anything in docs/previous-PR/issue/discussion??

Reply:
@GrayFlash I don't really aware of the docs about it, maybe @ravisuhag or @StewartJingga has more context with that.